Makefile

```diff
@@ -32,6 +32,10 @@ etcdir = $(prefix)/etc
 mandir = $(prefix)/share/man
 docdir = $(srcdir)/doc
 
+ifeq ($(DOCLANG), ko)
+  docdir = $(srcdir)/doc/ko
```

Comment:
Or `mandir` should have `/ko` at the end.

Reply:
I prefer to add `/ko` to `mandir` here only. Is it okay for you?

Reply:
Yep

Reply:
I checked it again, but it will be overriden by `mandir` in `.config` even if it's changed here.

Reply:
I tested it again and the system cannot find man page if it's under `ko` directory by default. It needs another way to read translated version. So I would like to leave it as is.

Reply:
Do you have other Korean man pages as well?  Did you set `LANG` env for `ko`?

Reply:
It still cannot find the page even after installing the man pages as follows:
```
$ pwd
/usr/local/share/man/man1/ko

$ ls
uftrace.1       uftrace-graph.1  uftrace-live.1    uftrace-recv.1    uftrace-report.1  uftrace-tui.1
uftrace-dump.1  uftrace-info.1   uftrace-record.1  uftrace-replay.1  uftrace-script.1

$ echo $LANG
ko_KR.UTF-8

$ man uftrace
No manual entry for uftrace
See 'man 7 undocumented' for help when manual pages are not available.
```
I don't have other Korean man pages in my system.

Reply:
I tested with `/usr/share/man/ko/man1`  and it worked well.

Reply:
It seems that my system reads the locale from `LC_ALL` first.
```
$ echo $LANG
ko_KR.UTF-8

$ echo $LC_ALL
en_US.UTF-8

$ realpath su.1.gz 
/usr/share/man/ko/man1/su.1.gz

$ uftrace -P. -la man ps
...
            [142588] | init_locale() {
  86.688 us [142588] |   setlocale(LC_ALL, "") = "en_US.UTF-8";
   6.413 us [142588] |   setenv("MAN_NO_LOCALE_WARNING", "1", 1) = 0;
   1.477 us [142588] |   bindtextdomain("man-db", "/usr/share/locale");
   0.883 us [142588] |   bindtextdomain("man-db-gnulib", "/usr/share/locale");
   1.004 us [142588] |   textdomain("man-db");
 101.381 us [142588] | } /* init_locale */
   0.780 us [142588] | setlocale(LC_MESSAGES, "NULL") = "en_US.UTF-8";
   1.127 us [142588] | strcmp("en_US.UTF-8", "POSIX") = 21;
   0.587 us [142588] | getenv("LANGUAGE") = "NULL";
            [142588] | xstrdup() {
   0.386 us [142588] |   strlen("en_US.UTF-8") = 11;
   0.443 us [142588] |   malloc(12) = 0x557e0b4eebc0;
   0.497 us [142588] |   memcpy(0x557e0b4eebc0, 0x557e0b4ee880, 12);
   4.486 us [142588] | } /* xstrdup */
...
```

Reply:
It cannot be set properly.
```
$ export LC_ALL=ko_KR.UTF-8
-bash: warning: setlocale: LC_ALL: cannot change locale (ko_KR.UTF-8)
```

Reply:
Did you set `LC_ALL` manually?

This is my case:
```
$ echo $LC_ALL
(nothing)

$ uftrace -la man uftrace
...
            [159234] | init_locale() {
 260.704 us [159234] |   setlocale(LC_ALL, "") = "ko_KR.UTF-8";
   1.968 us [159234] |   setenv("MAN_NO_LOCALE_WARNING", "1", 1) = 0;
   0.675 us [159234] |   bindtextdomain("man-db", "/usr/share/locale");
   0.255 us [159234] |   bindtextdomain("man-db-gnulib", "/usr/share/locale");
   0.237 us [159234] |   textdomain("man-db");
 266.200 us [159234] | } /* init_locale */
   0.385 us [159234] | setlocale(LC_MESSAGES, "NULL") = "ko_KR.UTF-8";
   0.407 us [159234] | strcmp("ko_KR.UTF-8", "POSIX") = 27;
   0.339 us [159234] | getenv("LANGUAGE") = "NULL";
...
   3.636 us [159234] |   openat(4, "/usr/share/man/ko/man1/uftrace.1", O_RDONLY) = 5;
...
```

Maybe you system lacks Korean locale support.  Please check system locale list.

```
$ locale -a | grep ko_KR
ko_KR.euckr
ko_KR.utf8
```

Reply:
> Did you set LC_ALL manually?

Yes, I did. So I removed the setting, but it also doesn't find the man page properly.
```
$ locale -a | grep ko_KR
ko_KR.utf8
```
I also tried to install with `apt install manpages-ko`, but it seems that Korean man pages are not well managed in Ubuntu. I couldn't find the proper `ppa`.

Reply:
Instead of considering locale right now, what about just install Korean man pages to the same directory as the current one? We may be able to properly update the locale when we find the proper way.

I think it may also be very difficult to someone else with similar troubles.

Reply:
Hmm.. never mind. I found the problem and it works fine now. I just updated the PR.